client/components/domains/search-filters/index.jsx

```diff
@@ -3,24 +3,25 @@
 /**
  * External dependencies
  */
-
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { pick, noop } from 'lodash';
 
 /**
  * Internal dependencies
  */
-
 import MoreFiltersControl from './more-filters';
+import TldFilterControl from './tld-filter';
 
 export default class SearchFilters extends Component {
 	static propTypes = {
 		filters: PropTypes.shape( {
 			includeDashes: PropTypes.bool,
 			maxCharacters: PropTypes.string,
 			showExactMatchesOnly: PropTypes.bool,
+			tlds: PropTypes.arrayOf( PropTypes.string ),
 		} ).isRequired,
+		availableTlds: PropTypes.arrayOf( PropTypes.string ),
```

Comment:
When there are TLDs in maintenance I get:
```
Failed prop type: Invalid prop `availableTlds` of type `object` supplied to `SearchFilters`, expected an array.
```

Looks like the backend indeed returns an object:
```
{
	"code": 200,
	"headers": [{
		"name": "Content-Type",
		"value": "application\/json"
	}],
	"body": {
		"1": "org",
		"2": "blog",
		// ...
	}
}
```